### DISCUSSION TITLE
Analyzing Simulations 'Element' attribute doesn't exist

### DISCUSSION THREAD

**Discussion by gruberanthony** [simnibs/simnibs#300] [9. Results and analysis] [answered]
I am attempting to do a batch analysis of the gray matter for simulations. However, when I run the example python script on the tutorial site it indicates the following error:

gray_matter=head_mesh.crop_mesh(simnibs.ElementTags.GM)
AttributeError: module 'simnibs' has no attribute 'ElementTags'

I have attempted to figure out why this is not working and have tried other methods of getting around this issue but they have also failed. Any assistance as to why this is occurring would be appreciated.

**Comment by oulap** [comment]
Which version of Simnibs are you using? The ElementTags were introduced in fairly recent version, so you might need to update your simnibs version.

**Reply by gruberanthony** [reply]
I am using version 4.0.0

**Comment by oulap** [comment]
Can you try to upgrade to v 4.0.1? You can download the installer here:

https://github.com/simnibs/simnibs/releases/tag/v4.0.1

This might also solve #301

**Reply by gruberanthony** [reply]
I am getting the same issue on version 4.0.1.

**Reply by oulap** [reply, marked as answer]
Okay, the problem is that we are releasing a new version (v4.1) very soon. The website is already updated, whereas the source code of the latest version is behind. The new version should hopefully be out next week. If you're in a rush, you can install the new version from source following the instructions here:

https://github.com/simnibs/simnibs?tab=readme-ov-file#installation-for-development
